package.json

```diff
@@ -36,6 +36,7 @@
     ]
   },
   "dependencies": {
+    "@metamask/scure-bip39": "^2.0.4",
```

Comment:
Shouldn't we remove `@scure/bip39` then too? 😅

Reply:
done [here](https://github.com/MetaMask/key-tree/pull/101/commits/24f59fae8a237638242988b3339a6e3e6b22b092)